app/controllers/advice_controller.rb

```diff
@@ -43,4 +43,10 @@ def save_advice
     end
     redirect_to action: 'edit_advice', id: params[:id]
   end
+
+  def question_advice_params(params_hash)
+    params_local = params
+    params_local[:@question_advice] = params_hash unless nil == params_hash
```

Comment:
Why did you use `:@question_advice` instead of `:question_advice` as the hash key?

Reply:
Sorry about that, it should be :question_advice instead of :@question_advice. Also, the :@team should be :team in advertise_for_partner_controller.rb. Do I need to modify it and push again?

Reply:
You're right, :@question_advice is not a good name here,  :question_advice is better. I just fixed it.